package.json

```diff
@@ -36,9 +36,12 @@
   "devDependencies": {
     "babel": "^5.2.16",
     "babel-core": "^5.2.6",
+    "babel-eslint": "^4.1.3",
     "babel-loader": "^5.0.0",
     "babel-plugin-object-assign": "^1.0.1",
     "brfs": "^1.2.0",
+    "eslint": "^1.7.3",
```

Comment:
This is useful as it doesn't require someone to install `eslint` globally to have linting work. It seems to be as much of one of the `devDependencies` any of the others in the list. Perhaps we should add linting to the npm scripts?

```json
"scripts": {
  "lint": "eslint"
}
```

and then:

```bash
npm run lint
```